packages/medusa/src/services/swap.ts

```diff
@@ -1221,6 +1216,33 @@ class SwapService extends TransactionBaseService {
       return result
     })
   }
+
+  protected async areReturnItemsValid(
+    returnItems: WithRequiredProperty<Partial<ReturnItem>, "item_id">[]
+  ): Promise<boolean> {
+    const manager = this.transactionManager_ ?? this.manager_
```

Comment:
thought: We've done this many places by now. Do you think it would make sense to create a method in the parent base service that always retrieves the correct manager?

I noticed @carlos-r-l-rodrigues added it in a separate class:
```
getManager() {
  return this.transactionManager_ ?? this.manager_
}
```

Reply:
Let me know what you think

Reply:
Sure, I think that's really neat 👍